config-connector/solutions/iam/kpt/pubsub-topic/README.md

```diff
@@ -0,0 +1,40 @@
+Pub/Sub Topic
+==================================================
+# NAME
+  pubsub-topic
+# SYNOPSIS
+  Config Connector compatible YAML files to grant a role to a particular IAM member for a PubSub topic.
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/):
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-topic .
```

Comment:
This works, but it might be better to explicitly specify the project name instead of relying on `.`.

Reply:
Let's be consistent and use `` `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-topic pubsub-topic` `` instead.

Reply:
Could you resolve the comments (not only this one) if you've fixed them in your PR?